### PR TITLE
Update event + profile property limits

### PR DESCRIPTION
**Pull request by kurbycchua**
Updated Event Property FAQ - minor textual clarification on event property names being 255 max length

Added User Property Limit in FAQ

### CODE REVIEW

File: pages/docs/data-structure/events-and-properties.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ We don't have a limit on the total number of events you can send to Mixpanel, bu
 
 We have a soft limit of 2000 distinct event names in a 30 day window. If you send more event names, we'll still ingest them, but those event names will not be indexed and will not appear in our autocomplete menus.
 
-Each event can have up to 2000 properties. Properties can be at most 255 characters in length (longer strings are truncated).
+Each event can have up to 2000 properties. Event property names can be at most 255 characters in length (longer strings are truncated).
```
Comment (mavlee) [inline pages/docs/data-structure/events-and-properties.md]: shouldn't this be values?

File: pages/docs/data-structure/user-profiles.md [hunk 2]
```diff
@@ -164,6 +164,11 @@ Mixpanel stores Events and User Profiles in two separate tables under the hood.
 ### How can I update User Profile Properties?
 User Profiles are mutable; Mixpanel only stores the latest value of each profile property. We have methods to update profile properties via our [HTTP API](https://developer.mixpanel.com/reference/profile-set).
 
+### What are the limits of User Profile Properties?
+Each User Profile can contain up to 2000 properties. User property names can be at most 255 characters in length (longer strings are truncated).
```
Comment (mavlee) [inline pages/docs/data-structure/user-profiles.md]: same as above

File: pages/docs/data-structure/events-and-properties.md (hunk 1 above)
Comment (kurbycchua) [reply pages/docs/data-structure/events-and-properties.md]: @mavlee it's the property name. only string data types has 255 but other data types have different limits which are already indicated here: https://docs.mixpanel.com/docs/data-structure/property-reference#supported-data-types

File: pages/docs/data-structure/user-profiles.md (hunk 2 above)
Comment (kurbycchua) [reply pages/docs/data-structure/user-profiles.md]: @mavlee same as above

**Comment by kurbycchua** [comment]
@mavlee let me know if the changes makes sense

File: pages/docs/data-structure/events-and-properties.md (hunk 1 above)
Comment (mavlee) [reply pages/docs/data-structure/events-and-properties.md]: i think it's probably worth either linking to that other section here, or adding that information somewhat redundantly

File: pages/docs/data-structure/events-and-properties.md [hunk 3]
```diff
@@ -59,6 +59,6 @@ We don't have a limit on the total number of events you can send to Mixpanel, bu
...
 
 **Note**: while events can have up to 2000 properties and batches of events can have up to 50 or 2000 events per batch depending on the ingestion endpoint, some libraries might default to sending the data through the GET protocol which would have a much lower size limit since the information is appended to the URL. If you are hitting the limits on the size of the request, first verify if the method being used is GET and can be switched to POST.
```
Comment (mavlee) [inline pages/docs/data-structure/events-and-properties.md]: i think we just changed the 50 to 2000, so maybe just update while we're here

File: pages/docs/data-structure/user-profiles.md (hunk 2 above)
Comment (mavlee) [reply pages/docs/data-structure/user-profiles.md]: same as above

File: pages/docs/data-structure/events-and-properties.md (hunk 3 above)
Comment (kurbycchua) [reply pages/docs/data-structure/events-and-properties.md]: updated

File: pages/docs/data-structure/events-and-properties.md (hunk 1 above)
Comment (kurbycchua) [reply pages/docs/data-structure/events-and-properties.md]: updated and linked.

File: pages/docs/data-structure/user-profiles.md (hunk 2 above)
Comment (kurbycchua) [reply pages/docs/data-structure/user-profiles.md]: updated and linked

**Review by marissakuhrau — APPROVED** [review]

**Pull request merged**